taxi/admin.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib import admin

```

Comment:
Exclude `__pycache__` directory with its content from the pull request for all apps.